LFGBulletinBoard/Options.lua

```diff
@@ -10,14 +10,17 @@ local PROJECT_EXPANSION_ID = {
 	[WOW_PROJECT_CLASSIC] = GBB.Enum.Expansions.Classic,
 	[WOW_PROJECT_BURNING_CRUSADE_CLASSIC] = GBB.Enum.Expansions.BurningCrusade,
 	[WOW_PROJECT_WRATH_CLASSIC] = GBB.Enum.Expansions.Wrath,
-	[WOW_PROJECT_CATACLYSM_CLASSIC] = GBB.Enum.Expansions.Cataclysm,
+	-- note: global not defined in classic era client
+	[WOW_PROJECT_CATACLYSM_CLASSIC or 0] = GBB.Enum.Expansions.Cataclysm,
 }
 local EXPANSION_PROJECT_ID = tInvert(PROJECT_EXPANSION_ID)
-local EXPANSION_NAME = {
-	[GBB.Enum.Expansions.Classic] = EXPANSION_NAME0,
-	[GBB.Enum.Expansions.BurningCrusade] = EXPANSION_NAME1,
+local EXPANSION_FILTER_NAME = {
+	[GBB.Enum.Expansions.Classic] = 
+		GBB.locales[GetLocale()]["PanelFilter"] or EXPANSION_NAME0,
+	[GBB.Enum.Expansions.BurningCrusade] = 
+		GBB.locales[GetLocale()]["TBCPanelFilter"] or EXPANSION_NAME1,
 	[GBB.Enum.Expansions.Wrath] = 
-		GBB.locales[GetLocale()]["WotlkPanelFilter"],
+		GBB.locales[GetLocale()]["WotlkPanelFilter"] or EXPANSION_NAME2,
```

Comment:
I think we should just the globals here instead of the custom localized strings.